### ISSUE TITLE
Product variation raw prices not visible for public users (not authenticated)

### ISSUE THREAD

**Issue by creative-andrew** [wp-graphql/wp-graphql-woocommerce#670]
**Describe the bug**
The raw product price variation comes as `null`  while querying them as a public user (not authenticated).

Query:
```
query NewQuery {
  products {
    nodes {
      ... on VariableProduct {
        id
        name
        variations {
          nodes {
            price(format: RAW)
            salePrice(format: RAW)
          }
        }
      }
    }
  }
}
```

Response:
```json
{
  "data": {
    "products": {
      "nodes": [
        {
          "id": "cHJvZHVjdDoxMg==",
          "name": "My variable product",
          "variations": {
            "nodes": [
              {
                "price": null,
                "salePrice": null
              },
              {
                "price": null,
                "salePrice": null
              },
              {
                "price": null,
                "salePrice": null
              }
            ]
          }
        }
      ]
    }
  }
}
```

**To Reproduce**
1. Create a variable product.
2. Set the prices for the variations.
3. Execute the query above.
4. See error: null on price.

**Expected behavior**
As a public user (not authenticated), I should be able to see the variations price formatted as RAW.
Right now this is only possible as a logged-in user. 

**Screenshots**
![image](https://user-images.githubusercontent.com/18492002/203530537-f0d01d53-0d82-40f5-99c7-2c03a54f90d3.png)

**Desktop (please complete the following information):**
 - OS: MacOs Monterry
 - Browser Chrome 
 - Version 107.0.5304.110 

**Plugin Versions**
- **WooGraphQL Version: 0.11.2
- **WPGraphQL Version: 1.12.2
- **WordPress Version: 6.11
- **WooCommerce Version: 7.1.0

**Comment by AVert** [comment]
Same

**Comment by kidunot89** [comment]
@creative-andrew @AVert This was resolved in `v0.12.0`
